lib/core/src/lib/card-view/components/card-view-selectitem/card-view-selectitem.component.html

```diff
@@ -1,36 +1,82 @@
+                }"
+            >{{ property.label | translate }}</div>
+            <div class="adf-property-field">
+                <div
+                    *ngIf="!isEditable"
+                    class="adf-property-value adf-property-read-only"
+                    [attr.data-automation-id]="'select-readonly-value-' + property.key"
+                    data-automation-class="read-only-value">{{ (property.displayValue | async) | translate }}
+                </div>
+                <div *ngIf="isEditable">
+                    <mat-form-field class="adf-property-value" [ngClass]="{'adf-property-value-editable': isEditable}">
+                        <mat-select
+                            [(value)]="value"
+                            [ngClass]="{ 'adf-property-readonly-value': isReadonlyProperty }"
+                            panelClass="adf-select-filter"
+                            (selectionChange)="onChange($event)"
+                            data-automation-class="select-box"
+                            [aria-label]="property.label | translate">
+                            <adf-select-filter-input *ngIf="showInputFilter" (change)="onFilterInputChange($event)"></adf-select-filter-input>
+                            <mat-option *ngIf="displayNoneOption">{{ 'CORE.CARDVIEW.NONE' | translate }}</mat-option>
+                            <mat-option
+                                *ngFor="let option of list$ | async"
+                                [value]="option.key">
+                                {{ option.label | translate }}
+                            </mat-option>
+                        </mat-select>
+                    </mat-form-field>
+                </div>
+            </div>
         </div>
-        <div *ngIf="isEditable">
-            <mat-form-field class="adf-property-value" [ngClass]="{'adf-property-value-editable': isEditable}">
-                <mat-select
-                    [(value)]="value"
-                    [ngClass]="{ 'adf-property-readonly-value': isReadonlyProperty }"
-                    panelClass="adf-select-filter"
-                    (selectionChange)="onChange($event)"
-                    data-automation-class="select-box"
-                    [aria-label]="property.label | translate">
-                    <adf-select-filter-input *ngIf="showInputFilter" (change)="onFilterInputChange($event)"></adf-select-filter-input>
-                    <mat-option *ngIf="displayNoneOption">{{ 'CORE.CARDVIEW.NONE' | translate }}</mat-option>
-                    <mat-option
-                        *ngFor="let option of list$ | async"
-                        [value]="option.key">
-                        {{ option.label | translate }}
+        <div *ngSwitchCase="'autocompleteBased'">
+            <mat-form-field
+                class="adf-property-field adf-card-selectitem-autocomplete"
+                [ngClass]="{
+                'adf-property-read-only': !isEditable
+            }"
```

Comment:
Looks like formatting is a little bit out of place here?

Reply:
right, fixed